librecad/src/main/lc_application.cpp

```diff
@@ -0,0 +1,24 @@
+#include <QStringList>
+#include <QFileOpenEvent>
+
+#include "lc_application.h"
+
+LC_Application::LC_Application(int &argc, char **argv)
+    : QApplication(argc, argv)
+{
+    QStringList files = QStringList();
```

Comment:
This is a local instance named `files`, this is not the `LC_Application::files` member variable!
So this line is useless.
`QStringList `'s constructor is called on creation of LC_Application, so initializing is not needed here.